apps/fulfillment-e2e/src/support/page_fragments/picking-lists.fragment.ts

```diff
@@ -1,5 +1,14 @@
 export class PickingListsFragment {
+  private getElement = (selector: string, withoutWrapper?: boolean) => {
+    return withoutWrapper ? cy.get(selector) : this.getWrapper().find(selector);
+  };
```

Comment:
This function looks strange. Do we really need it? Or maybe it can be refactored? (or maybe UI can be refactored so that we don't have to do this)

Reply:
The goal is to have the simplest page object (and tests possible)

Each additional method or function adds complexity and increases the costs of maintaining tests (which we do not want, right?)